### PR TITLE
Adds NameServers to abby.is-a.dev

### PR DESCRIPTION
**Pull request by MiloDev123**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/abby.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
         "discord": "699318362848886864"
     },
     "record": {
-        "CNAME": "mx.milosantos.com"
+        "NS": "ns1.milosantos.com",
+        "NS": "ns2.milosantos.com"
```
Comment (CuteDog5695) [inline domains/abby.json:9]:
```suggestion
        "NS": ["ns1.milosantos.com", "ns2.milosantos.com"]
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — DISMISSED** [review]
Fixed the file for you, now we have to wait for the admin to see this PR.

**Comment by MiloDev123** [comment]
I forgot to delete _discord.abby.is-a.dev, give me a second

**Comment by CuteDog5695** [comment]
> I forgot to delete _discord.abby.is-a.dev, give me a second

Oh, anyways

**Review by DEV-DIBSTER — DISMISSED** [review]

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by CuteDog5695** [comment]
@wdhdev If you want to see why user wants NS records, read the conversation at #17397.

**Pull request merged**